### DISCUSSION TITLE
Is the model getting better after using it?

### DISCUSSION THREAD

**Discussion by DavidMndz** [nextcloud/recognize#722] [Q&A] [answered]
Hello,

I can see that using Recognize in large sets of pictures is not very accurate, so I am wondering if the model gets updated and better after using it. Does it work in this way?

I think about correcting the faces as much as I can, removing photos from one person, merging others, etc. After that, if the model gets "more trained", I could remove all the faces and re-scan from scratch again, getting advantage from the previous training. Does this make any sense? 

If I am getting it wrong, I would appreciate some advice about how to train the model to get better results.

Thanks!

**Comment by marcelklehr** [comment, marked as answer]
> I could remove all the faces and re-scan from scratch again, getting advantage from the previous training. 

This is not the case. The detection model does is not trained online. The only "training" that happens is when you remove a face from a person, recognize will try to avoid readding the face to that same person again in the next run.

We are currently working on improving the face clustering. Stay tuned for updates.
